azurerm/data_source_dns_zone.go

```diff
@@ -43,6 +43,29 @@ func dataSourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
```

Comment:
can we update the documentation to include these new fields? This is kept in `./website/docs/d/dns_zone.html.markdown`

Reply:
done.